delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/files/model/FileStorageMetadata.scala

```diff
@@ -0,0 +1,16 @@
+package ch.epfl.bluebrain.nexus.delta.plugins.storage.files.model
+
+import akka.http.scaladsl.model.Uri
+import akka.http.scaladsl.model.Uri.Path
+import ch.epfl.bluebrain.nexus.delta.plugins.storage.files.model.FileAttributes.FileAttributesOrigin
+
+import java.util.UUID
+
+case class FileStorageMetadata(
```

Comment:
`FileStorage` ?

Reply:
It's not a file storage though, it's metadata regarding where the file is stored. FileStorage sounds too much like S3Storage etc

Reply:
I just would like to drop/replace the metadata suffix where we can